### PR TITLE
adding dw function example to mock when

### PR DESCRIPTION
**Pull request by pentreathm**
Adding example to mock-when

### CODE REVIEW

File: modules/ROOT/pages/mock-message-processor.adoc [hunk 1]
```diff
@@ -21,11 +21,30 @@ For example, you can use the Mock Event processor to mock a POST request with a
...
+----
+%dw 2.0
+output application/json
+---
+{
+	"foo" : "var"
+}
+----
+
+This Dataweave file creates the payload that you want to return in the POST and then use the `readUrl` DataWeave function to read the mapping file:
```
Comment (kevintroller) [inline modules/ROOT/pages/mock-message-processor.adoc]: Add method word...._that you want to return in the POST method and then use_

File: modules/ROOT/pages/mock-message-processor.adoc [hunk 2]
```diff
@@ -21,11 +21,30 @@ For example, you can use the Mock Event processor to mock a POST request with a
...
+
+[source,xml,linenums]
+----
+<munit-tools:then-return>
+  <munit-tools:payload value="#[readUrl('classpath://sample_data/mockPost.dwl')]" mediaType="application/json" encoding="UTF-8" />
+</munit-tools:then-return>
+----
+
+Finally you can configure a `then-return` element to define the type of response the mocked processor should return. It could be a payload, a variable, attributes, or even an error.
```
Comment (kevintroller) [inline modules/ROOT/pages/mock-message-processor.adoc]: Add , after Finally

**Review by kevintroller — APPROVED** [review]
Added some styling, the rest LGTM

File: modules/ROOT/pages/mock-message-processor.adoc (hunk 1 above)
Comment (fermujica) [reply modules/ROOT/pages/mock-message-processor.adoc]: You are right. Although I'd prefer to make it clear that this is a POST request. I think it's more straightforward than saying "the POST method of an HTTP request".
Change went here: 9d5fea8

**Pull request merged**